Formula/yaegi.rb

```diff
@@ -0,0 +1,26 @@
+class Yaegi < Formula
+  desc "Yet another elegant Go interpreter"
+  homepage "https://github.com/containous/yaegi"
+  url "https://github.com/containous/yaegi/archive/v0.7.2.tar.gz"
+  sha256 "1f2e63290614e26ffb82ef16af23d0ce5237c10a19a5c1191c22398adb62bc1e"
+  head "https://github.com/containous/yaegi.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
```

Comment:
We're working on removing the usage of GOPATH in formulae (see https://github.com/Homebrew/homebrew-core/issues/47627), will it be possible to build yaegi without it? 
As examples you could use linked to https://github.com/Homebrew/homebrew-core/issues/47627 PRs

Reply:
Thank you for informing me. I removed GOPATH and tested it again. It works flawlessly.